### DISCUSSION TITLE
Best Practices For Using DCE?

### DISCUSSION THREAD

**Discussion by keithadams** [Tyrrrz/DiscordChatExporter#701] [Q&A]
For most of the servers I want to archive, I have to use a self-bot from a second account. My ideal would  be one html file for each day of a channel. Are there any general recommendations for using DCE and flying under the radar?

For example:
1. Is it okay to repeat export once a day?
2. Is it better to export multiple channels at once or export single channels multiple times throughout the day?
3. Any approximate numbers on what's appropriate and what would be considered excessive? e.g. No more than `x` messages in a single call, no more than `y` number of commands a day, etc.

Thanks!

**Comment by Tyrrrz** [comment]
1. I think it depends on how large is the channel you are exporting. But in any case, I can't give you a definite answer because I don't know.
2. It doesn't matter.
3. No idea on that. I'd recommend being careful with the parallel limit option and keep it low (<=3).
